node/tests/ui_gateway_test.rs

```diff
@@ -169,7 +174,11 @@ fn cleanup_after_deceased_clients_integration() {
     let port = find_free_port();
     let mut node = utils::MASQNode::start_standard(
         "cleanup_after_deceased_clients_integration",
-        Some(CommandConfig::new().pair("--ui-port", &port.to_string())),
+        Some(
+            CommandConfig::new()
+                .pair("--chain", "polygon-mumbai")
```

Comment:
Maybe use the constant DEFAULT_TEST_CHAIN if there is one and if it is set to `polygon-mumbai`.

Reply:
DEFAULT_CHAIN used now